app/services/return-requirements/submit-check.service.js

```diff
@@ -28,6 +29,8 @@ async function go (sessionId) {
 
   await _validateLicence(session.licence.id)
 
+  await PersistReturnRequirementsService.go(session)
```

Comment:
We will need to guard against the user navigating to the previous check screen. As currently, if they go back and approve again, this will be a double insertion (a new record for each return requirement). 

We can add the id's in the session and do an update / upsert. I am not aware of the current practices in the system to know which way to go.